core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -747,6 +760,50 @@ object Boot extends IOApp with LazyLogging {
       openTelemetry
     )
   }
+
+  private def instantiateOpenTelemetry(conf: Config): OpenTelemetry = {
+    val maybeVersion = Option(getClass.getPackage.getImplementationVersion)
+    val resourceBuilder =
+      resources.Resource.getDefault.toBuilder
+        .put(ResourceAttributes.SERVICE_NAME, "rawls")
+    maybeVersion.foreach(version => resourceBuilder.put(ResourceAttributes.SERVICE_VERSION, version))
+    val resource = resourceBuilder.build
+
+    val maybeTracerProvider = conf.getBooleanOption("opencensus-scala.trace.exporters.stackdriver.enabled").flatMap {
+      case false => None
+      case true =>
+        val traceProviderBuilder = SdkTracerProvider.builder
+        val projectId = conf.getString("opencensus-scala.trace.exporters.stackdriver.project-id")
+        val googleTraceExporter =
+          TraceExporter.createWithConfiguration(TraceConfiguration.builder().setProjectId(projectId).build())
+        traceProviderBuilder.addSpanProcessor(BatchSpanProcessor.builder(googleTraceExporter).build())
+        val probabilitySampler =
+          Sampler.traceIdRatioBased(conf.getDouble("opencensus-scala.trace.sampling-probability"))
+        traceProviderBuilder
+          .setResource(resource)
+          .setSampler(Sampler.parentBased(probabilitySampler))
+          .build
+          .some
+    }
+
+//    val prometheusHttpServer = PrometheusHttpServer.builder().setPort(appConfig.prometheusConfig.endpointPort).build()
+//    system.registerOnTermination(prometheusHttpServer.shutdown())
+//    val sdkMeterProvider =
+//      SdkMeterProvider.builder
+//        .registerMetricReader(prometheusHttpServer)
+//        .setResource(resource)
+//        .build
```

Comment:
I am leaving these comments in here for now because I think we should switch to open telemetry for metrics reporting too

Reply:
it can be gradual with the statsd stuff. Unfortunately the old `OpenTelemetryMetrics` class from workbench-libs (which actually uses open census) is in use and sitting on the prometheus port